link/perf_event.go

```diff
@@ -136,10 +136,15 @@ func (pl *perfEventLink) Unpin() error {
 }
 
 func (pl *perfEventLink) Close() error {
+	if err := pl.fd.Close(); err != nil {
+		return fmt.Errorf("perf link close: %w", err)
+	}
+
+	// This will fail with EBUSY if the link above was a pinned bpf_link.
```

Comment:
From the code it seems like we don't support pinned perf events?

Reply:
based on the commit message, the comment looks good if `s/pinned/tracefs-backed`

Reply:
@lmb Perf events cannot be pinned, they are not bpf resources.
@mmat11 I'll make the comment a bit more elaborate, I really do mean 'pinned'. Closing a bpf_link's fd doesn't detach its event.